Tasks/a0_my_stack.py

```diff
@@ -31,8 +36,8 @@ def peek(ind: int = 0) -> Any:
 	:param ind: index of element (count from the top)
 	:return: peeked element
 	"""
-	print(ind)
-	return None
+	b = stack[1-ind]
```

Comment:
If I ask peek(1) you will check stack[0] instead of stack[-2], and this is not how it's upposed to be.
Blocking.